src/theme/components/alert.ts

```diff
@@ -0,0 +1,32 @@
+import { alertAnatomy } from '@chakra-ui/anatomy'
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+import { InfoIcon, WarningIcon, CheckIcon, CloseIcon } from '@chakra-ui/icons'
+
+const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(alertAnatomy.keys)
+
+const baseStyle = definePartsStyle({
+  container: {
+    bgColor: 'alert.bg.light',
```

Comment:
Alerts are also defined using colorSchemes. In this case, the colorScheme changes based on the status, being "blue" the default colorScheme (and status, since `info` is the default status).

You should style the expected color scheme instead.